app/controllers/films_controller.rb

```diff
@@ -59,7 +86,7 @@ def destroy
   private
 
```

Comment:
call before actin for show , edit , update , destory 

so before go inside the function you will have this ready 

def set_movie
    @movie=Film.find(params[:id])
end